.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "parserOptions": {
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "extends": ["airbnb-base"],
+  "rules": {
+    "no-shadow": "off",
+    "guard-for-in": "off",
+    "no-restricted-syntax": "off",
+    "no-param-reassign": "off",
+    "eol-last": "off",
+    "import/no-named-as-default": "off",
+    "import/no-named-as-default-member": "off",
+    "class-methods-use-this": "off"
+  },
```

Comment:
- [ ] Please ensure you use the same eslint rules as listed [here](https://github.com/microverseinc/linters-config/blob/master/javascript/.eslintrc.json).